src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateGoalTest.kt

```diff
@@ -238,6 +249,16 @@ class UpdateGoalTest : IntegrationTestBase() {
           }
           .wasCreatedAtPrison("BXI")
           .wasUpdatedAtPrison("MDI")
+          .wasCreatedBy("auser_gen")
```

Comment:
Assert the GoalEntity has the correct createdBy and updatedBy fields (which they would have done before, this was not the bug)